cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -42,6 +43,12 @@ bool gdv_fn_like_utf8_utf8(int64_t ptr, const char* data, int data_len,
   return (*holder)(std::string(data, data_len));
 }
 
+const char* gdv_fn_regexp_replace_utf8_utf8(int64_t ptr, const char* data, int data_len,
```

Comment:
The stub function needs to accept 2 more additional parameters, the **length** of the **replace_string** and the length of the output string.

Remember to do the changes in the line the add the function inside the GlobalMapping functions